clerk/jwks_cache.go

```diff
@@ -0,0 +1,40 @@
+package clerk
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"gopkg.in/square/go-jose.v2"
+)
+
+type jwksCache struct {
+	sync.Mutex
```

Comment:
I'd recommend to use a `sync.RWMutex` instead, since we want to be able to have multiple goroutines reading this value simultaneously.

Reply:
Nice catch